.github/actions/backport/action.yml

```diff
@@ -0,0 +1,73 @@
+name: Backport
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+  REPO_LOCATION:
```

Comment:
It's unclear what `REPO_LOCATION` is

Reply:
a14b9c75bf63357ca377f66d4c9168b2ce827c1a